Library/Formula/commandbox.rb

```diff
@@ -1,12 +1,12 @@
 class Commandbox < Formula
   desc "CFML embedded server, package manager, and app scaffolding tools"
   homepage "http://www.ortussolutions.com/products/commandbox"
-  url "http://downloads.ortussolutions.com/ortussolutions/commandbox/3.0.0/commandbox-bin-3.0.0.zip"
-  sha256 "194738f683cbbbaa63f7a2ae5d1fe6c0d09ef8370c4ac0b6c9aa087f6013abc1"
+  url "http://downloads.ortussolutions.com/ortussolutions/commandbox/3.0.1/commandbox-bin-3.0.1.zip"
+  sha256 "b5f821f5c327f5387a7cdf20855510fe39c1770dd2f0f383c4576519bae3d5d9"
 
   devel do
     url "http://integration.stg.ortussolutions.com/artifacts/ortussolutions/commandbox/3.1.0/commandbox-bin-3.1.0.zip"
-    sha256 "b2b9b74f7bd1f3b547d72e08e4b59c8c8163e6ba4504b2f46e8cf10884918351"
+    sha256 "37f2b7103d008daafa9f3ee279e701dcf9664fdcbde1e4da10fc85a8dd956819"
```

Comment:
From the other thread:

> The Jenkins build for stable rebuilt dev as well and overwrote the old file.  We're going to revise the build process a bit to prevent this, in the future.